### PR TITLE
[ONNX] Fix split export in opset13

### PR DESCRIPTION
**Pull request by KsenijaS**
Fix split export in opset13

### CODE REVIEW

File: torch/onnx/symbolic_opset13.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def split(g, self, split_size_or_sizes, dim, _outputs=None):
         return g.op("Split", self, split_size_or_sizes, axis_i=dim, outputs=_outputs)
     split_size = sym_help._get_const(split_size_or_sizes, 'i', 'split_size')
 
-    if self.type().sizes()[dim] is not None:
+    if self.type().sizes() is not None and self.type().sizes()[dim] is not None:
```
Comment (jiafatom) [inline torch/onnx/symbolic_opset13.py:73]:
```python
sizes = self.type().sizes()
if sizes is not None and sizes[dim] is not None
```

Comment (KsenijaS) [reply torch/onnx/symbolic_opset13.py:73]: good point. Thanks!

File: torch/onnx/symbolic_opset13.py [hunk 2]
```diff
@@ -70,7 +70,8 @@ def split(g, self, split_size_or_sizes, dim, _outputs=None):
         return g.op("Split", self, split_size_or_sizes, axis_i=dim, outputs=_outputs)
     split_size = sym_help._get_const(split_size_or_sizes, 'i', 'split_size')
 
-    if self.type().sizes()[dim] is not None:
+    sizes = self.type().sizes()
+    if sizes is not None and sizes[dim] is not None:
         size = self.type().sizes()[dim]
```
Comment (jiafatom) [inline torch/onnx/symbolic_opset13.py:75]: `size = sizes[dim]`

**Review by jiafatom — APPROVED** [review]

File: torch/onnx/symbolic_opset13.py [hunk 3]
```diff
@@ -70,8 +70,9 @@ def split(g, self, split_size_or_sizes, dim, _outputs=None):
         return g.op("Split", self, split_size_or_sizes, axis_i=dim, outputs=_outputs)
     split_size = sym_help._get_const(split_size_or_sizes, 'i', 'split_size')
 
-    if self.type().sizes()[dim] is not None:
-        size = self.type().sizes()[dim]
+    sizes = self.type().sizes()
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset13.py:73]: nit: can use `_get_tensor_dim_size` from `sym_help`.

**Review by BowenBao — APPROVED** [review]
LGTM, CI failure is not related, and should be resolved in current dev branch merge.
Will merge this PR after the dev branch merge.

**Pull request merged**